api.planx.uk/modules/auth/service.ts

```diff
@@ -1,4 +1,4 @@
-import { sign } from "jsonwebtoken";
+import jsonwebtoken from "jsonwebtoken";
```

Comment:
You're not doing anything wrong here or missing anything, this is just a CJS library without named exports.

See https://github.com/auth0/node-jsonwebtoken/pull/884

nit: `import jwt from "jsonwebtoken";` is a bit more terse and still meaningful?